package.json

```diff
@@ -16,7 +16,8 @@
 		"lodash": "4.17.21",
 		"popper.js": "1.16.1",
 		"sass": "1.62.1",
-		"select2": "4.0.13"
+		"select2": "4.0.13",
+		"yarn": "^1.22.19"
```

Comment:
I think if anything this is a dev dependency? If it wasn't here before it might just be a prerequisite?

Reply:
I can move it into the dev dependencies (?)